pkg/services/object/acl/v2/types.go

```diff
@@ -0,0 +1,29 @@
+package v2
+
+import (
+	"github.com/nspcc-dev/neofs-sdk-go/container"
+	cidSDK "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	eaclSDK "github.com/nspcc-dev/neofs-sdk-go/eacl"
+	"github.com/nspcc-dev/neofs-sdk-go/owner"
+)
+
+// ACLChecker is an interface that must provide
+// ACL related checks.
+type ACLChecker interface {
+	// BasicACLCheck must return true only if request
+	// passes basic ACL validation.
+	BasicACLCheck(RequestInfo) bool
+	// EACLCheck must return true only if request
+	// passes extended ACL validation.
+	EACLCheck(interface{}, RequestInfo) bool
+	// StickyBitCheck must return true only if sticky bit
+	// is disabled or enabled but request contains correct
+	// owner field.
+	StickyBitCheck(RequestInfo, *owner.ID) bool
+}
+
+// Classifier is an interface that must classify sender
+// via passed meta information, container and its ID.
+type Classifier interface {
+	Classify(MetaWithToken, *cidSDK.ID, *container.Container) (role eaclSDK.Role, isIR bool, ownerKey []byte, err error)
```

Comment:
Oh, too many return values, maybe structure them?

Reply:
Btw do we really need separate `Classifier` component? As I see all `ACLChecker` methods accept `RequestInfo`.

Reply:
made that private and deleted interface

structured return result